packages/google_workspace/changelog.yml

```
@@ -1,4 +1,15 @@
 # newer versions go on top
+- version: "1.8.1"
```

```suggestion
- version: "1.9.0"
```
because there is an enhancement in the list.